modules/mri_violations/help/resolved_violations.md

```diff
@@ -0,0 +1,3 @@
+# MRI Resolved Violations  
+
+This page displays scans that have been resolved from MRI Violation. Use the Selection Filter section to narrow down your search in the data results table below.  
```

Comment:
Could take another pass at this first sentence? There's something that doesn't read smoothly in that last few words.